pkgs/development/libraries/tracker/default.nix

```diff
@@ -50,11 +56,8 @@ stdenv.mkDerivation rec {
     gobject-introspection
     docbook-xsl-nons
     docbook_xml_dtd_45
-    python3 # for data-generators
-    systemd # used for checks to install systemd user service
-    dbus # used for checks and pkg-config to install dbus service/s
-  ] ++ checkInputs; # gi is in the main meson.build and checked regardless of
-                    # whether tests are enabled
+    (buildPackages.python3.withPackages (p: [ p.pygobject3 ]))
```

Comment:
can be switched to use pythonForBuild once https://github.com/NixOS/nixpkgs/pull/177888 is merged